gordo_components/builder/build_model.py

```diff
@@ -467,11 +469,13 @@ def provide_saved_model(
     replace_cache: bool
         Forces a rebuild of the model, and replaces the entry in the cache with the new
         model.
-    cv_mode: str
-        String which enables three different modes:
-        * cross_val_only: Only perform cross validation
-        * build_only: Skip cross validation and only build the model
-        * full_build: Cross validation and full build of the model, default value
+    evaluation_config: dict
+        Dict of parameters which are exposed to build_model.
+            - cv_mode: str
+                String which enables three different modes:
+                * cross_val_only: Only perform cross validation
+                * build_only: Skip cross validation and only build the model
+                * full_build: Cross validation and full build of the model, default value
```

Comment:
I think perhaps instead we should refer to the docs in `build_model`as done for the `data_config` and `model_config` parameters.